docs/source/specification.md

```diff
@@ -22,102 +27,147 @@ For example:
 
-Subject and session folder names cannot contain spaces and must consist of key-value pairs separated
-by underscores, e.g. `sub-001_id-5645332`. 
+* The `subject/session/datatype` folder structure within `derivatives` ***should***
+match that of `rawdata` whenever possible.
+
+* `subject` and `session` folder names ***must not*** contain spaces and ***must*** consist of key-value pairs separated by underscores, e.g. `sub-001_id-5645332`.
+
+* `datatype` folder names **must** be one of the following : `ephys`, `behav`, `funcimg`, `histology`.
+
+* If collected, `ephys`, `behav`, `funcimg` ***must*** be placed under the `session` level. If collected, `histology` ***must*** be placed under the `subject` level. 
+
+Below we describe each level of the `rawdata` folder hierarchy in more detail.
 
 ### Subject
-The subject level is prefixed with the *required* "sub" key - value pair, e.g. `sub-001`. Only one subject
-folder per-subject is permitted, and subject labels must be unique to each subject. *optional* key - value pairs 
-can be added after sub-<label> (e.g. `sub-001_id-5645332`).
+
+* Subject-level folders ***must*** be prefixed with a key-value pair that is unique for each subject. The key ***must*** be "sub" and the value ***must*** be numerical, e.g. `sub-001`. 
+* It is ***recommended*** that subjects are assigned ascending numerical labels as they are added to the project. The labels ***should*** be prefixed with an arbitrary number of 0s for consistent indentation, e.g. `sub-001`, `sub-002`, `sub-003`.
+* Each subject ***must*** have exactly one subject-level folder. 
+* Additional key-value pairs with alphanumerical labels ***may*** be appended after the "sub" key-value pair. For example, animal IDs (e.g. from the animal facility) can be added as follows: `sub-001_id-5645332`. The ***keys*** should be consistent across subjects.
+
+:::{hint}
+* valid: `sub-02`, `sub-001_id-5645332_sex-F`
+* invalid: `mouse-01`, `sub-001_female`, `sub-B`
+:::
 
 ### Session
-Within the subject directory reside session subdirectories named with the *required* prefix "ses", e.g. `ses-<label>`. A session represents a recording, 
-functional imaging or behavioural experimental session, and may contain multiple blocks, or runs, of data acquisition.
-Different sessions may have different combinations of datatypes. *optional* key - value pairs can be added after ses-<label>
-(e.g. `ses-001_date-220516`).
+
+* Session-level folders ***must*** be prefixed with a key-value pair that is unique for each session. The key ***must*** be "ses" and the value ***must*** be numerical, e.g. `ses-01`. 
+* It is ***recommended*** that sessions are assigned ascending numerical labels as they are added to the project. The labels ***should*** be prefixed with an arbitrary number of 0s for consistent indentation, e.g. `ses-01`, `ses-02`, `ses-03`.
```

Comment:
not 100% tied to required, recommended, optional as in some instances they do make thing a bit worried e.g. `a must do b` vs. `a is required to do b`. However I think we should pick 3 only as avoids having to remember what categories each word belongs to (for example recommended vs. should vs. may do not have obvious groupings to me).